fixtures/small/single_line_method_call_chain_expected.rb

```diff
@@ -1,3 +1,15 @@
 def blorp
   method_call(arg_a, arg_b).chained_call
 end
+
+This::Is::Some::SuperDuperLongConstantThatOnlyHasOneCallInTheChain.build(
+  reason: Because::ThisCouldCauseUsSome::SeriousProblemsIfWeTriedToMultilineIt::EventThoughItActuallyDoesExtendBeyondTheLimit
+)
+
+This::Is::Some::SuperDuperLongConstantThatOnlyHasOneCallInTheChain
+  .build(
+    reason: Because::IPutThisOnMultipleLines
+  )
+
+this.surprisingly.can_break
```

Comment:
how does this work if this line is very long and there's a comment after it? can we add a sample